katdal/ms_extra.py

```diff
@@ -863,7 +863,7 @@ def populate_pointing_dict(num_antennas, observation_duration, start_time, phase
     start_time : float
         Start time of observation, as a Modified Julian Date in seconds
     phase_center : array of float, shape (2,)
-        Direction of phase center, in ra-dec coordinates as 2-element array
+        Direction of phase center, in (az, el) coordinates as 2-element array (?)
```

Comment:
What does the (?) signify here?

Reply:
I'm still not sure... Ideally this should be an array of shape e.g. `(num_times, 2, num_antennas)` but a single phase center for all antennas and all times is definitely not it.